packages/core/upload/server/bootstrap.js

```diff
@@ -44,7 +44,9 @@ const createProvider = config => {
   try {
     provider = require(modulePath);
   } catch (err) {
-    throw new Error(`Could not load upload provider "${providerName}".`);
+    throw new Error(
+      `Could not load upload provider "${providerName}".\n${err.message}\n${err.stack}`
```

Comment:
Why do we expose the stack in the error message here?

Reply:
Because in the issue the user had a mistake in their code, but couldn't figure out what it was, because it only printed the message, without any hint what went wrong.

Reply:
Instead of doing this you rather create the error and overwrite its stack with the caught error it will be handled in a better way up the chain